<?xml version="1.0" encoding="UTF-8" ?>
<testsuite tests="3" failures="3" name="AreWeThereYetTest" time="0.062" errors="0" skipped="0">
  <properties>
    <property name="java.runtime.name" value="OpenJDK Runtime Environment"/>
    <property name="java.vm.version" value="11.0.14.1+1"/>
    <property name="sun.boot.library.path" value="C:\Program Files\Eclipse Adoptium\jdk-11.0.14.101-hotspot\bin"/>
    <property name="maven.multiModuleProjectDirectory" value="C:\Users\downw\Documents\NetBeansProjects\mooc-java-programming-i\part02-Part02_06.AreWeThereYet"/>
    <property name="java.vm.vendor" value="Eclipse Adoptium"/>
    <property name="java.vendor.url" value="https://adoptium.net/"/>
    <property name="guice.disable.misplaced.annotation.check" value="true"/>
    <property name="path.separator" value=";"/>
    <property name="java.vm.name" value="OpenJDK 64-Bit Server VM"/>
    <property name="sun.os.patch.level" value=""/>
    <property name="user.script" value=""/>
    <property name="user.country" value="US"/>
    <property name="sun.java.launcher" value="SUN_STANDARD"/>
    <property name="java.vm.specification.name" value="Java Virtual Machine Specification"/>
    <property name="user.dir" value="C:\Users\downw\Documents\NetBeansProjects\mooc-java-programming-i\part02-Part02_06.AreWeThereYet"/>
    <property name="java.vm.compressedOopsMode" value="Zero based"/>
    <property name="java.runtime.version" value="11.0.14.1+1"/>
    <property name="java.awt.graphicsenv" value="sun.awt.Win32GraphicsEnvironment"/>
    <property name="os.arch" value="amd64"/>
    <property name="java.io.tmpdir" value="C:\Users\downw\AppData\Local\Temp\"/>
    <property name="line.separator" value="
"/>
    <property name="java.vm.specification.vendor" value="Oracle Corporation"/>
    <property name="user.variant" value=""/>
    <property name="os.name" value="Windows 11"/>
    <property name="classworlds.conf" value="C:\Program Files\TMCBeans\java\maven\bin\..\bin\m2.conf"/>
    <property name="sun.jnu.encoding" value="Cp1252"/>
    <property name="java.library.path" value="C:\Program Files\Eclipse Adoptium\jdk-11.0.14.101-hotspot\bin;C:\WINDOWS\Sun\Java\bin;C:\WINDOWS\system32;C:\WINDOWS;C:\Program Files\Eclipse Adoptium\jdk-11.0.14.101-hotspot\bin;C:\Program Files\Eclipse Adoptium\jdk-11.0.14.9-hotspot\bin;C:\Program Files\Common Files\Oracle\Java\javapath;C:\Program Files (x86)\Common Files\Oracle\Java\javapath;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\WINDOWS\System32\OpenSSH\;C:\Users\downw\AppData\Local\Microsoft\WindowsApps;;."/>
    <property name="maven.conf" value="C:\Program Files\TMCBeans\java\maven\bin\../conf"/>
    <property name="jdk.debug" value="release"/>
    <property name="java.class.version" value="55.0"/>
    <property name="java.specification.name" value="Java Platform API Specification"/>
    <property name="sun.management.compiler" value="HotSpot 64-Bit Tiered Compilers"/>
    <property name="os.version" value="10.0"/>
    <property name="library.jansi.path" value="C:\Program Files\TMCBeans\java\maven\bin\..\lib\jansi-native"/>
    <property name="user.home" value="C:\Users\downw"/>
    <property name="user.timezone" value="America/New_York"/>
    <property name="java.awt.printerjob" value="sun.awt.windows.WPrinterJob"/>
    <property name="file.encoding" value="UTF-8"/>
    <property name="java.specification.version" value="11"/>
    <property name="user.name" value="downw"/>
    <property name="java.class.path" value="C:\Program Files\TMCBeans\java\maven\bin\..\boot\plexus-classworlds-2.6.0.jar"/>
    <property name="org.slf4j.simpleLogger.defaultLogLevel" value="error"/>
    <property name="java.vm.specification.version" value="11"/>
    <property name="sun.arch.data.model" value="64"/>
    <property name="sun.java.command" value="org.codehaus.plexus.classworlds.launcher.Launcher -Dfile.encoding=UTF-8 -T=1 -q clean install"/>
    <property name="java.home" value="C:\Program Files\Eclipse Adoptium\jdk-11.0.14.101-hotspot"/>
    <property name="user.language" value="en"/>
    <property name="java.specification.vendor" value="Oracle Corporation"/>
    <property name="awt.toolkit" value="sun.awt.windows.WToolkit"/>
    <property name="java.vm.info" value="mixed mode"/>
    <property name="java.version" value="11.0.14.1"/>
    <property name="java.vendor" value="Eclipse Adoptium"/>
    <property name="maven.home" value="C:\Program Files\TMCBeans\java\maven\bin\.."/>
    <property name="file.separator" value="\"/>
    <property name="java.version.date" value="2022-02-08"/>
    <property name="java.vendor.url.bug" value="https://github.com/adoptium/adoptium-support/issues"/>
    <property name="sun.io.unicode.encoding" value="UnicodeLittle"/>
    <property name="sun.cpu.endian" value="little"/>
    <property name="java.vendor.version" value="Temurin-11.0.14.1+1"/>
    <property name="sun.desktop" value="windows"/>
    <property name="sun.cpu.isalist" value="amd64"/>
  </properties>
  <testcase classname="AreWeThereYetTest" name="test1" time="0.047">
    <failure message="Something strange happened. It may be that &apos;class AreWeThereYet&apos; class&apos;s public static void main(String[] args) -method is missing 
or your program crashed due to an exception. More information java.util.NoSuchElementException: No line found" type="java.lang.AssertionError">java.lang.AssertionError: Something strange happened. It may be that &apos;class AreWeThereYet&apos; class&apos;s public static void main(String[] args) -method is missing 
or your program crashed due to an exception. More information java.util.NoSuchElementException: No line found
	at org.junit.Assert.fail(Assert.java:88)
	at AreWeThereYetTest.callMain(AreWeThereYetTest.java:50)
	at AreWeThereYetTest.test(AreWeThereYetTest.java:34)
	at AreWeThereYetTest.test1(AreWeThereYetTest.java:17)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.lang.Thread.run(Thread.java:829)
</failure>
  </testcase>
  <testcase classname="AreWeThereYetTest" name="test2" time="0">
    <failure message="Something strange happened. It may be that &apos;class AreWeThereYet&apos; class&apos;s public static void main(String[] args) -method is missing 
or your program crashed due to an exception. More information java.util.NoSuchElementException: No line found" type="java.lang.AssertionError">java.lang.AssertionError: Something strange happened. It may be that &apos;class AreWeThereYet&apos; class&apos;s public static void main(String[] args) -method is missing 
or your program crashed due to an exception. More information java.util.NoSuchElementException: No line found
	at org.junit.Assert.fail(Assert.java:88)
	at AreWeThereYetTest.callMain(AreWeThereYetTest.java:50)
	at AreWeThereYetTest.test(AreWeThereYetTest.java:34)
	at AreWeThereYetTest.test2(AreWeThereYetTest.java:22)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.lang.Thread.run(Thread.java:829)
</failure>
  </testcase>
  <testcase classname="AreWeThereYetTest" name="test3" time="0.015">
    <failure message="Something strange happened. It may be that &apos;class AreWeThereYet&apos; class&apos;s public static void main(String[] args) -method is missing 
or your program crashed due to an exception. More information java.util.NoSuchElementException: No line found" type="java.lang.AssertionError">java.lang.AssertionError: Something strange happened. It may be that &apos;class AreWeThereYet&apos; class&apos;s public static void main(String[] args) -method is missing 
or your program crashed due to an exception. More information java.util.NoSuchElementException: No line found
	at org.junit.Assert.fail(Assert.java:88)
	at AreWeThereYetTest.callMain(AreWeThereYetTest.java:50)
	at AreWeThereYetTest.test(AreWeThereYetTest.java:34)
	at AreWeThereYetTest.test3(AreWeThereYetTest.java:27)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.lang.Thread.run(Thread.java:829)
</failure>
  </testcase>
</testsuite>